common/v2/services/Contracts/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface Contract {
+  name: string;
+  networkId: string;
```

Comment:
Is this used in network object lookups? If so, it should be an enum or string literal union so it can use mapped or index types

Reply:
We'd have to factor in handling custom node type additions.  This pr doesn't change that param, so i'd say its out of scope for this PR